include/rocksdb/db.h

```diff
@@ -862,6 +862,29 @@ class DB {
     }
   }
 
+  // Batched MultiGet-like API that returns wide-column entities grouped
+  // by column families for each key. The input is a list of keys and
+  // PinnableWideColumnsCollections. For any given keys[i] (where 0 <= i <
+  // num_keys), results[i] will contain result for the ith key. Each result will
+  // be returned as PinnableWideColumnsCollection. PinnableWideColumnsCollection
+  // is a vector of PinnableWideColumnsBundle. Each PinnableWideColumnsBundle
+  // will contain a ColumnFamilyHandle pointer, status and PinnableWideColumns.
```

Comment:
It would be nice to explain the notion of attribute groups here a bit (since we don't have any other attribute group related API yet). E.g. we could mention that wide-column entities can be split into attribute groups, which are implemented using column families. Also, it might be a good idea to point out that `results` is an in/out parameter (the caller is expected to provide the column family handle, and RocksDB will populate the status and the result).

Reply:
Yeah, agree. Let me come up with something. 

Do you think adding `Experimental` until we have the APIs for write-path and iterators for attribute group would be a good idea?